src/main/java/org/candlepin/subscriptions/tally/billing/BillableUsageController.java

```diff
@@ -150,35 +149,45 @@ private BillableUsage produceHourlyBillable(BillableUsage usage) {
   }
 
   private BillableUsage produceMonthlyBillable(BillableUsage usage) {
-    log.debug("Processing monthly billable usage {}", usage);
-
-    Optional<Double> contractOptional = Optional.empty();
-    try {
-      contractOptional = contractsController.getContractCoverage(usage);
-    } catch (ExternalServiceException ex) {
-      if (usage.getSnapshotDate().isAfter(OffsetDateTime.now().minus(30, ChronoUnit.MINUTES))) {
-        log.warn(
-            "{} - Unable to retrieve contract for usage less than {} minutes old. Usage: {}",
-            ErrorCode.CONTRACT_NOT_AVAILABLE,
-            30,
-            usage);
-        return null;
-      } else {
-        log.error(
-            "{} - Unable to retrieve contract for usage older than {} minutes old. Usage: {}",
-            ErrorCode.CONTRACT_NOT_AVAILABLE,
-            30,
-            usage);
-        return null;
-      }
-    }
+    log.info(
+        "Processing monthly billable usage for orgId={} productId={} uom={} provider={}, snapshotDate={}",
+        usage.getOrgId(),
+        usage.getProductId(),
+        usage.getUom(),
+        usage.getBillingProvider(),
+        usage.getSnapshotDate());
```

Comment:
In log message it will be helpful to put usage.getBillingAccountId() as well since contracts side we are not logging anything and we send it as one of the query parameter.

Reply:
DONE